dev/Expander/Expander.vcxitems

```diff
@@ -24,11 +24,6 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)ExpanderAutomationPeer.cpp" />
   </ItemGroup>
   <ItemGroup>
-    <Page Include="$(MSBuildThisFileDirectory)Expander.xaml">
-      <UseVisualStyle>Any</UseVisualStyle>
-      <Version>RS1</Version>
-      <Type>DefaultStyle</Type>
-    </Page>
```

Comment:
The convention is to have the template in the `Expander.xaml`, with resources such as brushes in `Expander_themeresources.xaml`. I would revert this change.

Reply:
I was going with previous controls I worked on - they all had just one themeresources file (ContentDialog, Menuflyout etc) including the template itself. Having two files only splits files into two places so you have to search separately when updating styles. @teaP @StephenLPeters can you comment on this please?